### DISCUSSION TITLE
Easiest/Quickest way to run custom SQL querries ?

### DISCUSSION THREAD

**Discussion by bloor** [pocketbase/pocketbase#5238] [Q&A] [answered]
Sup?
Wanted to know if I'm approaching things right / optimal in my implementation.
I need to run an update /reset query like so `UPDATE posts SET views = 0` pretty often.

What is the recommended way to do this ? 
- a hook/script in `pb_hooks` that results in having and `/api/posts-reset` endpoint
- something else? 

Personally I would really love to have a SQL EXECUTE functionality in the dashboard enabled for admins. So I can run my queries directly there.

**Comment by pnmcosta** [comment, marked as answer]
When you say "pretty often" you mean on schedule?  

1. setup a cron job https://pocketbase.io/docs/go-jobs-scheduling/ (or JS)
2. instantiate a dao without events `dao := daos.New(app.Dao().DB())` (or JS)
3. have fun https://pocketbase.io/docs/go-database/#executing-queries

> a SQL EXECUTE functionality in the dashboard enabled for admins

Oh hell no! 😅
